tests/MongoDB.Driver.Core.Tests/Core/Operations/FindOperationTests.cs

```diff
@@ -186,6 +188,50 @@ public void constructor_should_throw_when_resultSerializer_is_null()
             argumentNullException.ParamName.Should().Be("resultSerializer");
         }
 
+        [Fact]
+        public void CreateFindCommandOperation_should_prefer_top_level_fields_over_modifiers()
+        {
+            var subject = new FindOperation<BsonDocument>(_collectionNamespace, BsonDocumentSerializer.Instance, _messageEncoderSettings)
+            {
+                Hint = "x_2",
+                Max = new BsonDocument("max", 5),
+#pragma warning disable 618
+                MaxScan = 7,
+#pragma warning restore 618
+                Min = new BsonDocument("min", 3),
+                ReturnKey = true,
+                ShowRecordId = true,
+#pragma warning disable 618
+                Snapshot = true,
+#pragma warning restore 618
+#pragma warning disable 618
+                Modifiers = new BsonDocument
+                {
+                    { "$hint", "x_1" },
+                    { "$max", new BsonDocument("max", 1) },
+                    { "$maxScan", 1 },
+                    { "$min", new BsonDocument("min", 1) },
+                    { "$returnKey", false },
+                    { "$showDiskLoc", false },
+                    { "$snapshot", false }
+                }
+#pragma warning restore 618
+            };
+
+            var result = subject.CreateFindCommandOperation();
```

Comment:
Our general rule is to use one blank line between each of the arrange/act/assert sections of the test, so add a blank line between 221 (the act) and 222 (the asserts).

Reply:
Done